docs/tools/validator.md

```diff
@@ -36,126 +36,91 @@ As such, there is no file uploading as part of the validation.
     This response indicates that your dataset is not BIDS compliant.
     Try following the suggestions listed to make the appropriate corrections.
 
-    ![Error](../assets/img//PEz9hbd.png)
+    ![Error](../assets/img/validator_error.png)
 
 !!! warning
 
     This response indicates that your dataset is BIDS compliant,
     but there are some non-critical problems
-    (such as optional fields missing, or differences between participants).
+    (such as recommended fields missing, or differences between participants).
 
-    ![Warning](../assets/img//Gqwc1q9.png)
+    ![Warning](../assets/img/validator_warning.png)
 
 !!! success "Success :rocket:"
 
     This response indicates that your folder contains a valid BIDS Dataset!
 
-    ![Valid](../assets/img/DPFVXOR.png)
+    <!-- ![Valid](../assets/img/DPFVXOR.png) -->
 
 ## Command Line Version
 
 ### Verifying a BIDS compliant data set
 
-After [installation with npm](https://github.com/bids-standard/bids-validator)
+After [installation with deno](https://bids-validator.readthedocs.io/en/latest/user_guide/command-line.html#using-the-command-line)
 using the CLI is relatively simple.
 Locate a bids data set similar to the one below:
 
 ```bash
-user@host:~/bids-examples$ tree sub001/
+user@host:~/bids-examples$ tree pet001
 
 sub001/
 ├── dataset_description.json
 ├── participants.json
 ├── participants.tsv
 ├── README
 └── sub-01
-    ├── ses-baseline
-    │   ├── anat
-    │   │   ├── sub-01_ses-baseline_T1w.json
-    │   │   └── sub-01_ses-baseline_T1w.nii.gz
-    │   └── pet
-    │       ├── sub-01_ses-baseline_pet.json
-    │       └── sub-01_ses-baseline_pet.nii.gz
-    └── ses-intervention
+    └── ses-01
         ├── anat
-        │   ├── sub-01_ses-intervention_T1w.json
-        │   └── sub-01_ses-intervention_T1w.nii.gz
+        │   ├── sub-01_ses-01_T1w.json
+        │   └── sub-01_ses-01_T1w.nii
         └── pet
-            ├── sub-01_ses-intervention_task-eyes_events.json
-            ├── sub-01_ses-intervention_task-eyes_events.tsv
-            ├── sub-01_ses-intervention_task-eyes_pet.json
-            └── sub-01_ses-intervention_task-eyes_pet.nii.gz
+            ├── sub-01_ses-01_trc-CIMBI36_pet.json
+            ├── sub-01_ses-01_trc-CIMBI36_pet.nii.gz
+            ├── sub-01_ses-01_trc-CIMBI36_recording-autosampler_blood.json
+            ├── sub-01_ses-01_trc-CIMBI36_recording-autosampler_blood.tsv
+            ├── sub-01_ses-01_trc-CIMBI36_recording-manual_blood.json
+            └── sub-01_ses-01_trc-CIMBI36_recording-manual_blood.tsv
 ```
 
-Now simply point the bids validator at the folder path of the subject(s) in
-question:
+Now simply point the bids validator at the folder path of the subject(s) in question:
 
 ```bash
-user@host:~/bids-examples$ bids-validator sub001
+user@host:~/bids-examples$ bids-validator pet001
```

Comment:
I would probably prefer to give an accurate command, though I can see the argument that simple might be better here:

```suggestion
user@host:~/bids-examples$ deno run -ERN jsr:@bids/validator pet001
```